<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZXingTest</class>
 <widget class="QWidget" name="ZXingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZXingTest</string>
  </property>
  <widget class="QLabel" name="id_qr_image">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_select">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_image_path">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="id_qr_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>二维码信息：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="id_text_info">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>310</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
